airflow/ui/src/components/DagRunInfo.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/jsx-boolean-value */
```

Comment:
I think we can drop `{ assumeUndefinedIsFalse: true }` from this in `rules/react.js`

Reply:
Removed the rule. Thanks.